sources/packages/backend/apps/test-db-seeding/src/test-db-seeding.module.ts

```diff
@@ -6,8 +6,9 @@ import { DiscoveryService } from "@golevelup/nestjs-discovery";
 import { CleanDatabase } from "./clean-db/clean-db";
 import {
   DesignationAgreementService,
+  CreateInstitutionsAndAuthenticationUsers,
   InstitutionUserService,
-} from "./db-seeding/02-institution";
+} from "./db-seeding/institution";
```

Comment:
did we change the pattern?

Reply:
The idea when it was done was to mimic the Cypress UI tests in place, which also were not following what seems to be a good naming convention either.
So, I do not have any argument to keep following those naming conventions. Does it make sense?

Reply:
okay, so, we will change changing the Cypress naming conventions too right

Reply:
Yes, I believe that we need to get rid os those numbers.